<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Braindead Password Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>User Details</string>
    </property>
    <widget class="QLineEdit" name="lineEditFirstName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelFirstName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
     <property name="buddy">
      <cstring>textEditFirstName
</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="labelLastName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
     <property name="buddy">
      <cstring>textEditFirstName
</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLastName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCompany">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelCompany">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Company</string>
     </property>
     <property name="buddy">
      <cstring>textEditFirstName
</cstring>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditKeywords">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelKeywords">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Keywords one per line</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="General">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QCheckBox" name="checkBoxCase">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>WIll transform all passwords case to upper,lower and title</string>
      </property>
      <property name="text">
       <string>case</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxPasswordJoin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Joins two passwords together.</string>
      </property>
      <property name="text">
       <string>Join passwords combo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPasswordJoinChars">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>1337</string>
     </attribute>
     <widget class="QCheckBox" name="checkBoxLeet">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Will leet just one letter in password with every combination.</string>
      </property>
      <property name="text">
       <string>Leet One</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxLeetAll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Will leet all leetable letters in password.</string>
      </property>
      <property name="text">
       <string>Leet All</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Prefix/Suffix</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditPrefixSymbols">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Comma seperated list of symbols to prefix each word.</string>
      </property>
      <property name="statusTip">
       <string>Comma seperated list of symbols to prefix each word.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prefix Symbols</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Suffix Symbols</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSuffixSymbols_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Comma seperated list of symbols to suffix each word.</string>
      </property>
      <property name="statusTip">
       <string>Comma seperated list of symbols to suffix each word.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Automaticall add incrementing numbers to end of short words to supplied length</string>
      </property>
      <property name="text">
       <string>Pad Suffix Length</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPadLength">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Automaticall add incrementing numbers to end of short words to supplied length</string>
      </property>
      <property name="statusTip">
       <string>Automaticall add incrementing numbers to end of short words to supplied length</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonSavePasswords">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Passwords</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Generate</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>9</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelGeneratedPasswords">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Generated Passwords</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSortPasswords">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sort passwords</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonCheckAll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Check All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUncheckAll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Uncheck All</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
